lib/srv/termmanager.go

```diff
@@ -53,7 +53,7 @@ type TermManager struct {
 	// remaining is a partially read chunk of stdin data
```

Comment:
Maybe: change `on bool` to `state termState` (and declare the accompanying enum). It's a bit more explicit (and a bit more flexible) than a bool.